src/libUtils/EvmCallParameters.h

```diff
@@ -15,18 +15,20 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-#ifndef ZILLIQA_SRC_LIBUTILS_RUNNERDETAILS_H_
-#define ZILLIQA_SRC_LIBUTILS_RUNNERDETAILS_H_
+#ifndef ZILLIQA_SRC_LIBUTILS_EVMCALLPARAMETERS_H_
+#define ZILLIQA_SRC_LIBUTILS_EVMCALLPARAMETERS_H_
 
 #include <string>
 
-struct RunnerDetails {
+// input parameters to Json call
+
+struct EvmCallParameters {
   std::string m_from;
   std::string m_to;
   std::string m_code;
   std::string m_data;
-  const uint64_t m_available_gas;
+  uint64_t m_available_gas;
   const boost::multiprecision::uint128_t m_balance;
```

Comment:
nit: either remove this const, or make other members const too.  I don't think part of this struct could be immutable, and part not.

Reply:
The old code used to use a parameter as both an in/out this has been changed so the are input parameters to the function, also balance is not currently passed through it is hard coded to send "00" for this field, currently the evm interface doesn't handle anything other than the "00" needs to be addressed,

Reply:
This is actually Steve who wishes to merge the changes onto evm_support_main, valeriy raised the PR on my behalf.